src/routes/directions.js

```diff
@@ -1,43 +1,83 @@
 const express = require('express');
-const Directions = require('../models/Directions')
-
+const Directions = require('../models/Directions');
 
 const router = express.Router();
+const directions = new Directions();
+
+// only expose these Google Directions API query parameters for now
+const validQueryParameters = ['origin', 'destination', 'mode', 'alternatives',
```

Comment:
@kurtishouser if you're interested, I would suggest refactoring these to use [sets](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set), the reason being that lookup with `someSet.has` is constant time whereas `someArray.includes` is linear.

Please consider this optional given your own current time constraints and that you are only working with a very small data set here.